.env.example

```
@@ -52,5 +52,6 @@ APPSMITH_RECAPTCHA_SITE_KEY=
 APPSMITH_RECAPTCHA_SECRET_KEY=
 APPSMITH_RECAPTCHA_ENABLED=
 
-# Intercom
-APPSMITH_DISABLE_INTERCOM=
+# Pylon in-app chat (optional runtime override; prefer REACT_APP_PYLON_APP_ID at build)
+APPSMITH_PYLON_APP_ID=
+APPSMITH_DISABLE_PYLON=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The APPSMITH_DISABLE_PYLON key should go before the APPSMITH_PYLON_APP_ID key

(UnorderedKey)